source/quickstart/container.rst

```diff
@@ -6,35 +6,158 @@ Quickstart for Containers
 
 .. default-domain:: minio
 
-This page deploys MinIO as a container for use with early development
-and evaluation environments.
+This page deploys a standalone MinIO deployment as a as a container with an attached persistent volume for use with early development and evaluation environments.
 
-For instructions on deploying to production environments,
-see :ref:`deploy-minio-distributed`.
+For instructions on deploying to production environments, see :ref:`deploy-minio-distributed`.
 
 Prerequisites
 -------------
 
-ToDo
+- `Podman <https://podman.io/getting-started/installation.html>`_ or `Docker <https://docs.docker.com/get-docker/>`_ installed
+- Read, write, and delete access to the folder or drive used for the persistent volume
 
 Procedure
 ---------
 
-#. Step 1
+Use the following commands to create the container depending on the container service and local operating system.
 
-   Contents of Step 1
+We provide GNU/Linux, MacOS, or Windows instructions for the following container types:
 
-#. Step 2
+- :ref:`quickstart-podman`
+- :ref:`quickstart-docker-root`
+- :ref:`quickstart-docker-regular-user`
 
-   Contents of Step 2
+.. _quickstart-podman:
 
-#. Step 3
+Podman
+~~~~~~
 
-   Contents of Step 3
+.. tab-set::
+
+   .. tab-item:: GNU/Linux or MacOS
+
+      .. code-block:: shell
+         :class: copyable
+
+         podman run \
+            -p 9000:9000 \
+            -p 9001:9001 \
+            -v ~/minio/data:/data \
+            quay.io/minio/minio server /data --console-address ":9001"
+
+   .. tab-item:: Windows
+
+      .. code-block:: shell
+         :class: copyable
+
+         podman run \
+            -p 9000:90900 \
+            -p 9001:9001 \
+            -v D:\data:/data \
+            quay.io/minio/minio server /data --console-address ":9001"
+
+.. _quickstart-docker-root:
+
+Docker (as ``root`` user)
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. tab-set::
+
+   .. tab-item:: GNU/Linux or MacOS
+
+      .. code-block:: shell
+         :class: copyable
+
+         mkdir -p ~/minio/data
+
+         docker run \
+            -p 9000:9000 \
+            -p 9001:9001 \
+            --name minio1 \
```

Comment:
we could probably call it `minio` - unless that for some reason conflicts with the default pod name?

I'm also unsure the name matters if we aren't running as detatched